Apps/WebClient/src/ClientApp/src/components/timeline/linearTimeline.vue

```diff
@@ -120,10 +121,54 @@ export default class LinearTimelineComponent extends Vue {
             self.setPageFromDate(eventDate);
         });
 
+        this.eventBus.$on(
+            EventMessageName.TimelineCurrentDateUpdated,
+            function (currentDate: Date) {
+                self.goToPageByCurrentDate(currentDate);
+            }
+        );
+
         window.addEventListener("resize", this.handleResize);
         this.handleResize();
     }
 
+    private goToPageByCurrentDate(currentDate: Date) {
+        const selectedYearMonth = moment(currentDate).format("YYYY-MM");
+        let currentPage = 1;
+        let currentEntryIndex = 1;
+        let i = 0;
+        let foundLastEntryOfSelectedMonth = false;
+        const timelineEntriesLenght = this.timelineEntries.length;
+        // scan from the most recent entries to the older ones
+        while (i < timelineEntriesLenght && !foundLastEntryOfSelectedMonth) {
```

Comment:
Instead of using a while with an index, a for is preferred

Reply:
how to do you break the "for"? I have tried foreach but breaking it is not recommended.

Reply:
I would write it like:
```
for (let i = 0; i< i < timelineEntriesLenght && !foundLastEntryOfSelectedMonth; i++)
```

Reply:
sounds good, I will make the changes